styles/style.ts

```diff
@@ -0,0 +1,24 @@
+import { Styles } from 'styled-components';
+import theme from './theme';
+
+const style: Styles = {
+	btnStyle: {
+		white_btn: {
+			width: `${200}px`,
+			backgroundColor: `${theme.colors.white}`,
+			color: `${theme.colors.gray_007}`,
+			border: `1px solid ${theme.colors.gray_004}`,
+		},
+		blue_btn_001: {
+			width: `${200}px`,
+			backgroundColor: `${theme.colors.primary_003}`,
+			color: `${theme.colors.white}`,
+		},
+		blue_btn_002: {
+			width: `${240}px`,
+			backgroundColor: `${theme.colors.primary_003}`,
+			color: `${theme.colors.white}`,
+		},
+	},
+};
+export default style;
```

Comment:
결과적으로는 이번 PR 에서 button 컴포넌트 스타일링을 위한 방법으로 style 객체로 관리하는 것을 채택이 된 상황인가봅니다

고려해볼만한 상황으로는, 사용하는 컴포넌트 내부적으로 지금 위에 선언된 스타일링 사이에 변화가 state 에 따라 변경이 필요할 경우 분기 처리 같은 로직이 들어갈 것이라 판단이 듭니다.

추가적으로, 우리는 css-in-js 방식을 기본적으로 사용하기로 했으나 현 시점으로 파악해보자면 우리가 스타일링을 하는 방식이 여러가지 인 것 같습니다.

1. css-in-js 스타일링 컴포넌트 
2. (샐리가 이번 PR 에 제시한) style 객체
3. (아직 PR 올리진 않았지만 빅터가 적용한, because. createPortal 을 통한 Modal 내 스타일 컴포넌트 적용이 안되고 있는 << PR 에서 좀 더 구체적으로 언급하겠음) GlobalStyle 에 styled class 접근자를 통한 스타일링
4. (3번과 더불어) inline styling

이렇게 파악이 되네요 ~!

뭐, 이런 방식이 `나쁜 것` 은 아니라 생각하지만, 이곳저곳 분산이 되어있다보니 관리 측면에서는 장기적인 프로젝트라고 가정했을 경우, 효율적인 방식은 아니라고 누군가는 생각할 수 있을 것 같습니다.

위에 언급한 3,4 번의 경우는 현재 해결을 하진 못했지만, 차선책을 선택을 해서라도 분산된 스타일링 방식을 최대한 한가지 방향으로 통일할 생각이니 참고 부탁드립니다

Reply:
> 3. (아직 PR 올리진 않았지만 빅터가 적용한, because. createPortal 을 통한 Modal 내 스타일 컴포넌트 적용이 안되고 있는 << PR 에서 좀 더 구체적으로 언급하겠음) GlobalStyle 에 styled class 접근자를 통한 스타일링
> 4. (3번과 더불어) inline styling 

> 위에 언급한 3,4 번의 경우는 현재 해결을 하진 못했지만, 차선책을 선택을 해서라도 분산된 스타일링 방식을 최대한 한가지 방향으로 통일할 생각이니 참고 부탁드립니다

아, 혹시나 NextJS (현재, 13 버전 기준에서의 발생하는 문제이긴하나) 에서 createPortal 을 통한 Modal 컴포넌트 내에서만 특이하게 css-in-js 방식의 스타일링이 적용이 안되는 현상을 경험해봤거나, 이를 해결하기 위한 레퍼런스를 발견하게 된다면, 적극적인  공유 부탁드립니다 ~! 🥹